src/Pyz/Zed/DynamicEntity/data/installer/configuration.json

```diff
@@ -821,29 +832,26 @@
                     "isCreatable": true,
                     "isEditable": true,
                     "type": "integer",
-                    "validation": {
-                        "isRequired": true
-                    }
+                    "validation": { "isRequired": true }
                 },
                 {
-                    "fieldName": "fk_resource_product_abstract",
-                    "fieldVisibleName": "fk_resource_product_abstract",
+                    "fieldName": "url",
+                    "fieldVisibleName": "url",
                     "isCreatable": true,
                     "isEditable": true,
-                    "type": "integer",
+                    "type": "string",
                     "validation": {
-                        "isRequired": false
+                        "isRequired": true,
+                        "constraints": [{ "name": "url" }]
                     }
                 },
                 {
-                    "fieldName": "url",
-                    "fieldVisibleName": "url",
+                    "fieldName": "fk_resource_product_set",
+                    "fieldVisibleName": "fk_resource_product_set",
                     "isCreatable": true,
                     "isEditable": true,
-                    "type": "string",
-                    "validation": {
-                        "isRequired": true
-                    }
```

Comment:
Smth went wrong, please check https://github.com/spryker/suite-nonsplit/blob/master/src/Pyz/Zed/DynamicEntity/data/installer/configuration.json (and fix for all the demoshop PRs)